src/SDKs/CognitiveServices/data-plane/Vision/Microsoft.CognitiveServices.Vision.Tests/Microsoft.CognitiveServices.Vision.Tests.csproj

```diff
@@ -0,0 +1,35 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>Microsoft.CognitiveServices.Vision.Tests</PackageId>
+    <Description>Microsoft.CognitiveServices.Vision.Tests Class Library</Description>
+    <AssemblyName>Microsoft.CognitiveServices.Vision.Tests</AssemblyName>
+    <VersionPrefix>1.0.0-preview</VersionPrefix>
+  </PropertyGroup>
+  <PropertyGroup>    
+    <TargetFramework>netcoreapp1.1</TargetFramework>
+  </PropertyGroup>
+  
+  <ItemGroup>
+  <PackageReference Include="Microsoft.Azure.Management.ResourceManager" Version="1.6.0-preview" />
+    <ProjectReference Include="..\Microsoft.CognitiveServices.Vision\Microsoft.CognitiveServices.Vision.csproj" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Update="SessionRecords\**\*.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+  </ItemGroup>
+  <ItemGroup>
+    <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Update="TestImages\detection1.jpg">
```

Comment:
@DavidLiCIG combine items in Itemgroup starting line 27 to be part of ItemGroup starting line 17
Please organize items in limited item groups especially if they logically belong to same category